src/services/expenses.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+const { v4: uuidv4 } = require('uuid');
+
+let expenses = [];
+
+function getAllExpenses(requestQuery) {
+  let filteredExpenses = expenses;
+
+  const { userId, categories, from, to } = requestQuery;
+
+  if (userId) {
+    filteredExpenses = filteredExpenses.filter(expense => (
+      expense.userId === +userId
```

Comment:
Use the `Number()` function instead of the unary plus operator to convert a string to a number. Replace `+userId` with `Number(userId)`.